helm/portieris/templates/webhooks.yaml

```diff
@@ -29,4 +28,5 @@ webhooks:
         apiVersions: ["*"]
         resources: ["pods", "deployments", "replicationcontrollers", "replicasets", "daemonsets", "statefulsets", "jobs", "cronjobs"]
     failurePolicy: Fail
-{{ end }}
+    sideEffects: None
+    admissionReviewVersions: ["v1beta1"]
```

Comment:
These look fine but can we double-check by running the e2e tests against this branch?